src/clone-npm-pkg.js

```diff
@@ -0,0 +1,37 @@
+const path = require('path');
+const os = require('os');
+const crypto = require('crypto');
+const shell = require('shelljs');
+const promiseSpawn = require('./promise-spawn.js');
+const exitHook = require('exit-hook');
+
+const run = function(cwd) {
```

Comment:
This logic used to live in the main `verify` function, but it just made things confusing.